resources/lib/play_utils.py

```diff
@@ -405,8 +405,8 @@ def play_file(play_info):
         if user_data.get("PlaybackPositionTicks") != 0:
 
             reasonable_ticks = int(user_data.get("PlaybackPositionTicks")) / 1000
-            seek_time = reasonable_ticks / 10000
-            display_time = str(timedelta(seconds=seek_time))
+            seek_time = round(reasonable_ticks / 10000,0)            
```

Comment:
```suggestion
            seek_time = round(reasonable_ticks / 10000,0)
```

Big trailing whitespace here